chandra_aca/maude_decom.py

```diff
@@ -310,7 +312,7 @@ def _aca_image_msid_list(pea):
     np.array([PIXEL_MAP_INV["4x4"][f"{k}1"] for k in _a2p]).T,
     np.array([PIXEL_MAP_INV["6x6"][f"{k}1"] for k in _a2p]).T,
     np.array([PIXEL_MAP_INV["6x6"][f"{k}2"] for k in _a2p]).T,
-    [],
+    np.array([PIXEL_MAP_INV["4x4"][f"{k}1"] for k in _a2p]).T,
```

Comment:
What does this change end up doing?

Reply:
Ping @javierggt for this question from @jeanconn .

Reply:
My cursory reading was that this was part of "Deal with the fact that sometimes IMGTYP can be 3 (code DNLD, "memory download")" by making more 4x4 images for that case instead of an empty list, but things were a little opaque in this part of the code so I figured I'd check.

Reply:
ah, I missed the original question.

if image type is DNLD, the integer value is 3, which pulls that entry, which is empty in the master version. That ends up causing and exception. Wit this change DNLD images are treated just like 4x4 images.

Reply:
Thanks!